Tests/Data/Parabolic/Richards/infiltration_1D/1d-infiltration-RM.prj

```diff
@@ -0,0 +1,315 @@
+                <property>
+                    <name>reference_temperature</name>
+                    <type>Constant</type>
+                    <value>293.15</value>
+                </property>
+                <property>
+                    <name>relative_permeability</name>
+                    <type>RelativePermeabilityVanGenuchten</type>
+                    <exponent>0.358974358974359</exponent>
+                    <minimum_relative_permeability_liquid>1e-3</minimum_relative_permeability_liquid>
+                    <residual_liquid_saturation>0.18139535</residual_liquid_saturation>
+                    <residual_gas_saturation>0.0</residual_gas_saturation>
+                </property>
+                <property>
+                    <name>saturation</name>
+                    <type>SaturationVanGenuchten</type>
+                    <residual_liquid_saturation>0.18139535</residual_liquid_saturation>
+                    <residual_gas_saturation>0.0</residual_gas_saturation>
+                    <exponent>0.358974358974359</exponent>
+                    <p_b>2725</p_b>
+                </property>
+                <property>
+                    <name>bishops_effective_stress</name>
+                    <type>BishopsPowerLaw</type>
+                    <exponent>1</exponent>
+                </property>
+            </properties>
+        </medium>
+    </media>
+    <time_loop>
+        <processes>
+            <process ref="Hus">
+                <nonlinear_solver>NLS</nonlinear_solver>
+                <convergence_criterion>
+                    <type>PerComponentDeltaX</type>
+                    <norm_type>NORM2</norm_type>
+                    <abstols>1e-8 1e-16 1e-16</abstols>
+                </convergence_criterion>
+                <time_discretization>
+                    <type>BackwardEuler</type>
+                </time_discretization>
+                <time_stepping>
+                    <type>IterationNumberBasedTimeStepping</type>
+                    <t_initial>0.0</t_initial>
+                    <t_end>86400</t_end>
+                    <initial_dt>0.1</initial_dt>
+                    <minimum_dt>0.1</minimum_dt>
+                    <maximum_dt>100</maximum_dt>
+                    <!-- linear function m=1.25-0.025*n omitting m=1, because of stale situation -->
+                    <number_iterations>1 2 3 4 5 6 7 8 9 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25</number_iterations>
+                    <multiplier> 1.225 1.2 1.175 1.15 1.125 1.1 1.075 1.05 1.025 0.975 0.95 0.925 0.9 0.875 0.85 0.825 0.8 0.775 0.75 0.725 0.7 0.675 0.65 0.625</multiplier>
+                </time_stepping>
+            </process>
+        </processes>
+        <output>
+            <type>VTK</type>
+            <prefix>1d-infiltration-RM</prefix>
+            <suffix>_t_{:gtime}</suffix>
+            <timesteps>
+                <pair>
+                    <repeat>1</repeat>
+                    <each_steps>10000</each_steps>
```

Comment:
@waltherm Change this to
```xml
<repeat>100000</repeat>
<each_steps>1</each_steps>
```
for full output.

Reply:
Yes, I set this before to have output during a fixed time stepping at specific times.